src/helpers/authManager.js

```diff
@@ -418,6 +422,8 @@ const updateRedirectUri = (aMgr, sRedirectUri) => {
 
 export const login = (bFullReauth=false) => {
 
+  processQueryParams();
```

Comment:
Are we calling this function twice for both embedded and portal?

Reply:
We're doing it to capture query parameters specified at different instances.